### DISCUSSION TITLE
My gist has disappeared

### DISCUSSION THREAD

**Discussion by rpuig** [community/community#35775] [Other Feature Feedback, Questions, & Ideas]
I created a gist like 3 months ago, and now ists nowhere in my gists page, and i did not delete it. Is it possible that someone deleted it from github admins? 

Its the gist i created to sync my settings with Settings sync extension for Microsoft Visual Code

**Comment by jfullstackdev** [comment]
posting it here may increase visibility, particularly for the Staff, they might respond but there is no assurance

another one you can try is the Support

https://support.github.com/contact/account
